Lib/test/support/__init__.py

```diff
@@ -1982,3 +1983,13 @@ def skip_if_broken_multiprocessing_synchronize():
             synchronize.Lock(ctx=None)
         except OSError as exc:
             raise unittest.SkipTest(f"broken multiprocessing SemLock: {exc!r}")
+
+
+def check_disallow_instantiation(testcase, tp, *args, **kwds):
```

Comment:
If you want, you can document it at: https://docs.python.org/dev/library/test.html#module-test.support

Reply:
Will do!